airflow/bin/cli.py

```diff
@@ -259,39 +259,63 @@ def pools(args):
     :param args: parsed args object with a property of filepath to airflow pools yaml file
     :return: None
     """
+    session = settings.Session()
     with open(args.filepath, 'r') as stream:
         try:
             pools_config = yaml.load(stream)
         except yaml.YAMLError:
+            print("Failed to load: {}".format(args.filepath))
```

Comment:
Not directly related to your PR, but just noticed that we swallow this exception. Should we? (I don't see why we should)

Reply:
yeah we should pop the error up.

Reply:
Sometimes it's nice to swallow exception on the CLI and print nice statements instead.

Reply:
Yeah. `exit(n)` where `n` is a value greater than 0 will do that though.